bin/codeclimate

```diff
@@ -4,5 +4,6 @@ $LOAD_PATH.unshift(File.expand_path(File.join(File.dirname(__FILE__), "../lib"))
 require "cc/cli"
 
 ENV["FILESYSTEM_DIR"] ||= "."
+ENV["CODE_PATH"] ||= ENV["PWD"]
```

Comment:
This was a change I had to make to get things working in my local dev environment (where I run everything within a Docker container. Since it's an `||=` it seemed safe enough to commit)